webpack.config.js

```diff
@@ -29,6 +29,18 @@ const path = require('path');
         test: /\.css$/i,
         use: ['style-loader', 'css-loader'],
       },
+      {
+        test: /\.(gif|png|jpe?g)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[name].[ext]',
+              outputPath: 'assets/images/',
+            },
+          },
+        ],
+      },
```

Comment:
- [ ] I dug further and made research, I was able to figure that this is part of what is not making the the Js file to load, so I checked webpack documentation and I reaslised that the correct usage is 
```
{
        test: /\.(png|svg|jpg|jpeg|gif)$/i,
        type: 'asset/resource',
      },
```
so I replaced it with that, but the image is still not loading but the js file now loads, I would implore you to dig further into that.